### ISSUE TITLE
Role sync for offline members

### ISSUE THREAD

**Issue by TheShrubber** [thatsmybis/thatsmybis#307]
As of right now, roles assigned to members in discord only updates on TMB if they log into the website.  The problem we have is that if a member gets demoted that change won't be reflected on TMB unless we ask that person to also log into TMB purely for the roles to sync.  We plan on using the attendance feature moving forward and pairing that with raid teams being set up on the site to reflect the rank they have in the guild/discord.

Simply put I'd like an option if it's technically possible to sync the roles of offline members based off of what roles they currently have on discord.

I'm sorry if this has been requested before, and if it is not possible for some technical reason/cost I understand.  Thanks!
